ui-permissions/RM-super-user-permissions.sql

```diff
@@ -0,0 +1,17 @@
+insert into casev3.user_group
+values ('8269d75c-bfa1-4930-aca2-10dd9c6a2b42', 'Super user - full permissions', 'super');
+
+insert into casev3.user_group_permission
+values ('c469377e-680e-4cb1-92a0-5217be2b3a52', -- permission ID
+        'SUPER_USER',                           -- activity
+        '8269d75c-bfa1-4930-aca2-10dd9c6a2b42', -- group ID
+        null);
+
+insert into casev3.users
+values ('8d44dca6-359e-4e84-b55d-2d1903f9e561', -- user ID
+        'gavin.edwards@ons.gov.uk');                    -- user email
```

Comment:
Should this be moved to the top and gavin.edwards@ons.gov.uk replaced with something like 
insert into casev3.users
values ('8d44dca6-359e-4e84-b55d-2d1903f9e561', -- user ID
        <REPLACE WITH ONS EMAIL>);                    -- user email

That way it would fail if the user didn't replace it with at least a valid string - hopefully their own ons email.   If it failed it would be easier to run again as it should bomb out on the 1st command